Frameworks/CoreFoundation/URL.subproj/CFURL.h

```diff
@@ -406,6 +406,9 @@ CFStringRef CFURLCreateStringByReplacingPercentEscapesUsingEncoding(CFAllocatorR
 CF_EXPORT
 CFStringRef CFURLCreateStringByAddingPercentEscapes(CFAllocatorRef allocator, CFStringRef originalString, CFStringRef charactersToLeaveUnescaped, CFStringRef legalURLCharactersToBeEscaped, CFStringEncoding encoding) CF_DEPRECATED(10_0, 10_11, 2_0, 9_0, "Use [NSString stringByAddingPercentEncodingWithAllowedCharacters:] instead, which always uses the recommended UTF-8 encoding, and which encodes for a specific URL component or subcomponent (since each URL component or subcomponent has different rules for what characters are valid).");
 
+// WINOBJC : An additional method for using character sets has been added for efficiency.
```

Comment:
(There's no need for passive voice.0

---
In reply to: [77097076](https://github.com/Microsoft/WinObjC/pull/946#discussion_r77097076) [](ancestors = 77097076)